.github/workflows/ci.yml

```diff
@@ -95,7 +97,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ env.RUST_VERSION }}
-          targets: "thumbv6m-none-eabi,wasm32-unknown-unknown"
+          targets: "thumbv6m-none-eabi,riscv32imac-unknown-none-elf,wasm32-unknown-unknown"
```

Comment:
Just curious - did you chose the IMAC extensions for any particular reason, or is that just a safe default?

Reply:
That's closes to the internal target used by risc-zero